tests/Doctrine/DBAL/Migrations/Tests/Finder/RecursiveRegexFinderTest.php

```diff
@@ -63,6 +63,7 @@ public function testFindMigrationsReturnsTheExpectedFilesFromDirectory()
             '20150502000004' => 'TestMigrations\\Version20150502000004',
             '20150502000005' => 'TestMigrations\\Version20150502000005',
             '1_reset_versions' => 'TestMigrations\\Version1_reset_versions',
+            '1_symlinked_file' => 'TestMigrations\\Version1_symlinked_file',
```

Comment:
I'd suggest adding a whole new test (and set of fixture files) for the new feature here.